project/BuildHelper.scala

```diff
@@ -26,8 +26,8 @@ object BuildHelper {
   val Scala213: String   = versions("2.13")
   val ScalaDotty: String = "3.1.0" //versions.getOrElse("3.0", versions("3.1"))
 
-  val zioVersion        = "1.0.12"
-  val zioJsonVersion    = "0.2.0-M2"
+  val zioVersion        = "2.0.0-RC1"
+  val zioJsonVersion    = "0.3.0-RC1-1"
   val zioPreludeVersion = "1.0.0-RC7"
```

Comment:
Do you want to update the ZIO Prelude and ZIO Optics versions as well?

Reply:
Yes, wasn't aware of the releases there. Thanks!